media_driver/agnostic/gen12/codec/hal/codechal_vdenc_vp9_g12.cpp

```diff
@@ -972,6 +972,11 @@ MOS_STATUS CodechalVdencVp9StateG12::GetSystemPipeNumberCommon()
     {
         disableScalability = userFeatureData.i32Data ? true : false;
     }
+    else
+    {
+	// WA disable Scalability on G12 for VP9 encoding by default.
+	disableScalability = true;
+    }
```

Comment:
suggested change to:
//bool disableScalability = m_hwInterface->IsDisableScalability();
//bool disableScalability = true;
if ......

Reply:
Updated